synedrion/Cargo.toml

```diff
@@ -50,9 +51,24 @@ k256 = { version = "0.13", default-features = false, features = ["ecdsa", "arith
 impls = "1"
 hex = { version = "0.4", default-features = false, features = ["alloc"] }
 test-log = { version = "0.2.16", default-features = false, features = ["trace", "color"] }
+tracing-subscriber = "0.3.19"
+
+[features]
+private_benches = ["criterion"]
 
 [[bench]]
 bench = true
 name = "bench"
 harness = false
 path = "benches/bench.rs"
+
+[[bench]]
+bench = true
+name = "pow"
+harness = false
+path = "benches/pow.rs"
```

Comment:
Why does this one use `path`, but the other doesn't?

Reply:
Copy&paste. I don't think it's needed. Fixing.

Reply:
Actually I removed the `pow` bench. I'd like to keep it and similar "investigation-only" benches but need a think on where to put them.